README.html

```diff
@@ -20,117 +20,199 @@
-            ones.<br>
-            So, it is strictly advised <b>not</b> to use it directly on your host machine without any virtual machine or
-            docker.<br>
-            Do not host code from this project on internet-facing servers.<br><br>
+            provided to learn about:</p>
+        <ul>
+            <li>The vulnerability</li>
+            <li>How it can be exploited</li>
+            <li>The part of the code causing this vulnerability.</li>
+            <li>How to patch this vulnerability at different levels.</li>
+        </ul>
+
+        <h2>Installation</h2>
+        <ol>
+            <li>
+                To set up this application on Windows, you must run Apache and MySQL services, which are inbuilt in
+                XAMPP.
+            </li>
+            <li>
+                Follow this for setting up Cyber Sentinel on Windows.
+            </li>
+            <li>
+                After setting up and downloading the repository, rename the folder to cybersentinel and place it inside
+                htdocs
+            </li>
+            <li>
+                You can also download the dockerized version of the application here, which will require you to download
+                the Docker Desktop application for Windows.
+            </li>
+            <li>
+                Follow the steps in this Docker Hub repository to use this application on Docker.
+            </li>
+
+        </ol>
+
+
+
+
+
+        <h2>For users</h2>
+        <ul>
+            <li>
+                There are different vulnerabilities with different difficulty levels. The difficulty levels are based on
+                how good the patch is for that specific vulnerability.
+            </li>
+            <li>
+                There is no fixed objective to complete a module. If you feel you've exploited the system thoroughly,
+                the goal is reached!
+            </li>
+            <li>
+                There is a help button at the bottom to view hints & tips for that vulnerability. There are additional
+                links for further reading on each vulnerability.
+            </li>
+        </ul>
+
+
+        <h2>For developers</h2>
+        <ul>
+            <li>
+                Refer to CONTRIBUTING.md and DEVELOPMENT.md for more information on how to contribute to this project.
+            </li>
+        </ul>
+
+        <h2 style="text-align: center;">Disclaimer</h2>
+        <p style="text-align: center;">This application is vulnerable! There are documented vulnerabilities that will
+            cause more undocumented ones.
+            So, it is strictly advised not to use it directly on your host machine without any virtual machine or
+            docker.
+            Do not host code from this project on internet-facing servers.</p>
+        <br />
+        <p style="text-align: center;">This application is vulnerable! There are documented vulnerabilities that will
```

Comment:
"This application is vulnerable! There are documented vulnerabilities that will ".

This line is added unnecessarily in lines 183, and 191.

Reply:
![image](https://github.com/SuchitReddi/cybersentinel/assets/91052168/dfefdda0-e903-4a94-b3a2-817b9fe1978e)
its there in your main branch already , look once

Reply:
![image](https://github.com/SuchitReddi/cybersentinel/assets/72243194/8273b463-ada0-45d4-926b-827790a81e25)
It is there only in the first paragraph, but in your file, it is present in the beginning of all paragraphs.